examples/compiler_selection/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("@aspect_rules_sol//sol:defs.bzl", "sol_binary")
+
+# TODO: The user already told us about the solidity version in their source file with a pragma.
+# It's a pity to make them repeat it here, and then provide no error checking if they get it wrong.
+# It would be neat for sol_binary to have a validation action that can tell the user about the
+# compiler_version attribute, if the toolchain doesn't support what we find in the file.
+# Maybe it could even suggest a buildozer command to fixup.
```

Comment:
I think the user should be explicit about which version they want, so I'm happy (even keen) to exclude pragma inspection.

As Ethereum smart contracts are immutable, best practice would require that the compiler version never changes otherwise the local build will no longer reflect what's on-chain.

Reply:
sounds like a TODO for a later PR, right?